crates/ra_assists/src/assists/raw_string.rs

```diff
@@ -15,6 +16,39 @@ pub(crate) fn make_raw_string(mut ctx: AssistCtx<impl HirDatabase>) -> Option<As
     ctx.build()
 }
 
+pub(crate) fn make_raw_string_unescaped(mut ctx: AssistCtx<impl HirDatabase>) -> Option<Assist> {
```

Comment:
It's to late here for me to be able to actually think, so this might be horriblty wrong, but

Why is this a separate code action? I would thing "make string raw", "make string unraw" are the only two we need?

Reply:
because of this comment: https://github.com/rust-analyzer/rust-analyzer/issues/1730#issuecomment-524282273

Reply:
Did I misunderstood? ^^'